components/brave_wallet_ui/common/async/lib.ts

```diff
@@ -568,7 +571,9 @@ export function refreshKeyringInfo () {
     dispatch(WalletActions.setDefaultAccounts(filteredDefaultAccounts))
 
     // Get selectedAccountAddress
-    const getSelectedAccount = await keyringService.getSelectedAccount(selectedCoin)
+    const getSelectedAccount = selectedCoin === BraveWallet.CoinType.FIL
+        ? await keyringService.getFilecoinSelectedAccount(selectedNetwork.chainId)
```

Comment:
This will fail if the `selectedNetwork` is not a `Filecoin` `chainId`.
Since each `coinType` has a `defaultNetwork`, you need to get `defaultNetworks` from state and then pass in the `chainId` for the `defaultNetwork` of `coinType` FIL.

Reply:
Scratch that, was confused for a moment.
This is good!